crates/bitwarden/src/vault/collection.rs

```diff
@@ -13,28 +13,28 @@ use crate::{
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 #[cfg_attr(feature = "mobile", derive(uniffi::Record))]
 pub struct Collection {
-    id: Option<Uuid>,
-    organization_id: Uuid,
+    pub id: Option<Uuid>,
+    pub organization_id: Uuid,
 
-    name: EncString,
+    pub name: EncString,
 
-    external_id: Option<String>,
-    hide_passwords: bool,
-    read_only: bool,
+    pub external_id: Option<String>,
+    pub hide_passwords: bool,
+    pub read_only: bool,
```

Comment:
The long term aspiration is to not expose this model externally. So I'm a bit hesitant to just make all of these public with that knowledge.

Would it not be sufficient if we verify that the decrypted view matches our expectation?